nilearn/glm/contrasts.py

```diff
@@ -200,7 +200,7 @@ def __init__(
         variance : array of shape (n_voxels)
             The associated variance estimate.
 
-        dim : int or None, optional
+        dim : :obj:`int` or None, optional
```

Comment:
OK let's forget about the "optional" in the methods for now otherwise these PRs will never end!

Reply:
Just to clarify, `or None` means that the default value is `None`, correct? 
I agree; there are too many changes to be made, such as these. I can try to do this in another sprint, though. I think my feet are too deep into this issue now, haha.